woocommerce/wc-admin
# Add pre-fill for email address field in OBW

Fixes #7517

This PR adds a prefill capability to the email address field in OBW. It sources the email from Jetpack if it's available and defaults to WordPress' admin email if not.

### Detailed test instructions:

- Reset the onboarding data. You can do this with [woocommerce-admin-test-helper](https://github.com/woocommerce/woocommerce-admin-test-helper), go to Tools > WCA Test Helper > Tools > Reset Onboarding Wizard.
- Make sure Jetpack is connected. I used WCPay setup to force a reconnect after manually disconnecting. 
- Go to OBW store details.
- Observe the email field is pre-filled with your WordPress.com's email address (Do not click on continue as this will persist the email in profile option).
- Disconnect from Jetpack, you can do this using WCA test helper tools or manually in Plugins > Installed plugins > Disconnect and deactivate Jetpack.
- Go to OBW store details.
- Observe the email field is pre-filled with your WordPress' admin email address.
- Change the email to something else, like `test@asd.com` and click continue to persist it to profile option.
- Go back to store details and refresh the page.
- Observe that the email address is persisted.

No changelog because this is going to a feature branch
